plugins/UFM_NDT_Plugin/build/config/cfg_files_merge.py

```diff
@@ -0,0 +1,114 @@
+"""
```

Comment:
already discussed - we said that we want to copy this merger module from external shared location for all the plugins (not a copy for each plugin) - can you please handle that?

Reply:
implementing and testing